### PR TITLE
Upgrade Jitsi SDK to 6.2.2

### LINKED ISSUE #5745: Viewing Jitsi participant crashes the app
Probably since Jitsi 5.0.2, incuded in Element 1.4.10

- Start a Jitsi call and join the conf
- Click on the three dot menu and on "Participants"

The app crashes.

On our side you can maybe disable this menu entry, but I guess the issue has to be fixed on the Jitsi library.

Stacktrace:
```
java.lang.NullPointerException: Attempt to invoke virtual method 'boolean com.facebook.react.uimanager.FabricViewStateManager.hasStateWrapper()' on a null object reference
        at com.facebook.react.views.textinput.ReactEditText.updateCachedSpannable(ReactEditText.java:981)
        at com.facebook.react.views.textinput.ReactEditText.access$300(ReactEditText.java:75)
        at com.facebook.react.views.textinput.ReactEditText$TextWatcherDelegator.onTextChanged(ReactEditText.java:1092)
        at android.widget.TextView.sendOnTextChanged(TextView.java:11631)
```

### PR DESCRIPTION
**Pull request by bmarty**
## Type of change

- [ ] Feature
- [ ] Bugfix
- [ ] Technical
- [x] Other : lib upgrade

## Content

Upgrade Jitsi to 6.2.2.
As a bonus, I have added a debug activity to display some info about Jitsi. I did not find a way to retrieve the version and both button does nothing. I keep it though, could be useful for future tests.

image

This upgrade should also fix this warning from the PlayStore:

image

(https://support.google.com/faqs/answer/12577537)

Demo of the new version:

- new pre-launch screen
- new Foreground service

Jitsi6 2 2_demo

## Motivation and context

Closes #5745 

## Tests

Crash is confirmed fixed.

But I have issue when trying to answer/reject a WebRTC call. Still investigating if it's new or not :/
*EDIT*: everything is working fine today.

## Tested devices

- [x] Physical
- [ ] Emulator
- OS version(s):

## Checklist

- [ ] Changes has been tested on an Android device or Android emulator with API 21
- [ ] UI change has been tested on both light and dark themes
- [ ] Accessibility has been taken into account. See https://github.com/vector-im/element-android/blob/develop/CONTRIBUTING.md#accessibility
- [ ] Pull request is based on the develop branch
- [ ] Pull request includes a new file under ./changelog.d. See https://github.com/vector-im/element-android/blob/develop/CONTRIBUTING.md#changelog
- [ ] Pull request includes screenshots or videos if containing UI changes
- [ ] Pull request includes a [sign off](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off)
- [ ] You've made a self review of your PR
- [ ] If you have modified the screen flow, or added new screens to the application, you have updated the test [UiAllScreensSanityTest.allScreensTest()](https://github.com/vector-im/element-android/blob/main/vector/src/androidTest/java/im/vector/app/ui/UiAllScreensSanityTest.kt#L73)

### CODE REVIEW

**Review by ganfra — APPROVED** [review]
One question otherwise LGTM

File: dependencies_groups.gradle [hunk 1]
```diff
@@ -65,6 +65,7 @@ ext.groups = [
                         'com.fasterxml.woodstox',
                         'com.gabrielittner.threetenbp',
                         'com.getkeepsafe.relinker',
+                        'com.giphy.sdk',
```
[the remark this answers is not in the document]
Comment (ganfra) [reply dependencies_groups.gradle:68]: I hate transitive dependencies in lib like that, but yay

**Review by yostyle — APPROVED** [review]
LGTM

**Comment by bmarty** [comment]
To be updated, exodus is detecting a new tracker "GIPHY Analytics":

image

File: dependencies_groups.gradle (hunk 1 above)
[the remark this answers is not in the document]
Comment (bmarty) [reply dependencies_groups.gradle:68]: And apparently exodus does not like this new dep as well :/

**Comment by bmarty** [comment]
Decision has been taken today to wait for Jitsi to provide a version without a tracker. So wait for https://github.com/jitsi/jitsi-meet/issues/11759 to be fixed and released.

**Comment by bmarty** [comment]
According to https://github.com/jitsi/jitsi-meet/issues/11759#issuecomment-1191316437 it seems that this could be unblocked.

**Comment by bmarty** [comment]
Updating to 6.2.2, but the code is not compiling yet.

File: dependencies_groups.gradle (hunk 1 above)
[the remark this answers is not in the document]
Comment (saghul) [reply dependencies_groups.gradle:79]: You can exclude it and the LIBRE_BUILD compile time flag was used the SDK will work without it.

[the remark this answers is not in the document]
Comment (bmarty) [reply dependencies_groups.gradle:79]: Yes, thanks, I will do it, I wanted to make the code to compile first.

**Review by ganfra — APPROVED** [review]

**Pull request merged**